NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/CountryTableCell.cs

```diff
@@ -24,8 +24,8 @@ public CountryTableCell (IntPtr handle) : base (handle)
 
         public void UpdateCell(CountryDetailsViewModel country)
         {
-            StyleUtil.InitLabelWithSpacing(CountryNameLabel, StyleUtil.FontType.FontRegular, country.Name, 1.14, 16, 28);
-            CheckboxImage.Image = (country.Checked ? UIImage.FromBundle("CheckboxChecked") : UIImage.FromBundle("CheckboxUnchecked"));
```

Comment:
Is CheckboxUnchecked already removed from assets? if not and it is not used anywhere else can you also remove it?

Reply:
Done